content/docs/deployment/csmoperator/modules/observability.md

```diff
@@ -14,30 +14,40 @@ The CSM Observability module for supported Dell CSI Drivers can be installed via
-  - Scenario 1: Deploy one supported CSI Driver and enable Observability module
-    - If you enable `metrics-powerscale` or `metrics-powerflex`, must enable `otel-collector` as well.
-    
-  - Scenario 2: Deploy multiple supported CSI Drivers and enable Observability module
-    - When deploying the first driver, enable all components of Observability module in the CR. 
-    - For the following drivers, only enable the metrics service, and remove `topology` and `otel-collector` sections from the CR.
-    - The CR created at first must be deleted at last.
-    - 
-Note: pods in the `karavi` namespace will be in the ContainerCreating state until certificates are successfully created as described in the next step.
+- Observability will deploy with self-signed certificates by default. If you want to have custom certificates created instead, please generate certificates and private keys, encode them in base64, and insert them into the sample file as shown below for whichever components you are enabling:
+```
+    # observability: allows to configure observability
+    - name: observability
+...
+      components:
+        - name: topology
+...
+          # certificate: base64-encoded certificate for cert/private-key pair -- add cert here to use custom certificates
+          #  for self-signed certs, leave empty string
+          # Allowed values: string
+          certificate: "<INSERT BASE64-ENCODED TOPOLOGY CERTIFICATE HERE>"
+          # privateKey: base64-encoded private key for cert/private-key pair -- add private key here to use custom certificates
+          #  for self-signed certs, leave empty string
+          # Allowed values: string
+          privateKey: "<INSERT BASE64-ENCODED TOPOLOGY PRIVATE KEY HERE>"
+...
+        - name: otel-collector
+...
+          # certificate: base64-encoded certificate for cert/private-key pair -- add cert here to use custom certificates
+          #  for self-signed certs, leave empty string
+          # Allowed values: string
+          certificate: "<INSERT BASE64-ENCODED OTEL-COLLECTOR CERTIFICATE HERE>"
+          # privateKey: base64-encoded private key for cert/private-key pair -- add private key here to use custom certificates
+          #  for self-signed certs, leave empty string
+          # Allowed values: string
+          privateKey: "<INSERT BASE64-ENCODED OTEL-COLLECTOR PRIVATE KEY HERE>"
+...
+```
+- Notes:
+    - If you enable `metrics-powerscale` or `metrics-powerflex`, you must enable `otel-collector` as well.
+    - otel-collector cannot be enabled without a metrics component also enabled.
+    - If you are deploying multiple drivers, only enable topology, otel-collector, and cert-manager in the first driver. For subsequent drivers, only enable the metrics component. When deleting the deployment, the driver that was created first must be deleted last.
```

Comment:
Should this be bolded or highlighted? Seems to be quite important to know.

Reply:
I can bold it, sure

Reply:
@AronAtDell what exactly were you hoping to bold?

Reply:
Start with the "Notes:" Then highlight "For subsequent drivers, only enable the metrics component." Then have "When deleting the deployment, the driver that was created first must be deleted last." as a separate bullet.